lib/elixir/lib/system.ex

```diff
@@ -53,7 +53,7 @@ defmodule System do
   the time, all calculations are done in the `:native` unit, to
   avoid loss of precision, with `convert_time_unit/3` being
   invoked at the end to convert to a specific time unit like
-  milliseconds or microseconds. See the `t:time_unit/0` type for
+  `millisecond` or `microsecond`. See the `t:time_unit/0` type for
```

Comment:
If we're using backticks here I guess we should use the full atom, no? Like `:millisecond` or `:microsecond`.